lighthouse-cli/test/fixtures/source-maps/source-maps-tester.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<!--
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+-->
+
+<html>
+<head>
+  <title>Source maps tester</title>
+  <meta charset="utf-8">
+</head>
+<body>
+  <!-- Note: none of these source maps are _actually_ maps of the 'generated' code here. -->
```

Comment:
would it help if they were?

http://sourcemapper.qfox.nl/ had a few examples of some

Reply:
Not at this stage. Certainly will be needed later.

Reply:
I wanted to actually generate the source maps (and not just check valid source maps in source), but it seemed more work than necessary for now, since here we are just gathering. I think the next part will be adding an audit for validating source maps, and at that point actual source maps will be of benefit.

so, in short, just deferring until needed, but yes it should be done.

Reply:
it seems trivial to do and much better than checking in filler placeholders